.github/workflows/active_update_truth_weekly.yml

```diff
@@ -80,13 +80,13 @@ jobs:
       env:
         GITHUB_PAT: ${{ secrets.GH_TOKEN }}
 
-    - run: mkdir ./covidData/data-raw/JHU/
```

Comment:
I suspect that we may still need this line here if we are using the command just below to install covidData.  But maybe we should just use `remotes::install_github` to install covidData now.  I think that would be preferable at this point...